pkg/cmd/server/origin/admission/plugin_initializer.go

```diff
@@ -138,25 +137,30 @@ func NewPluginInitializer(
 		}
 	}
 	// note: we are passing a combined quota registry here...
-	genericInitializer, err := initializer.New(kubeClientGoClientSet, informers.GetClientGoKubeInformers(), authorizer)
-	if err != nil {
-		return nil, nil, err
-	}
+	genericInitializer := initializer.New(
+		kubeClientGoClientSet,
+		informers.GetClientGoKubeInformers(),
+		authorizer,
+		legacyscheme.Scheme,
+	)
 	kubePluginInitializer := kadmission.NewPluginInitializer(
 		kubeInternalClient,
-		kubeExternalClient,
 		informers.GetInternalKubeInformers(),
-		authorizer,
 		cloudConfig,
 		restMapper,
-		quotaRegistry)
-	// upstream broke this, so we can't use their mechanism.  We need to get an actual client cert and practically speaking privileged loopback will always have one
-	kubePluginInitializer.SetClientCert(privilegedLoopbackConfig.TLSClientConfig.CertData, privilegedLoopbackConfig.TLSClientConfig.KeyData)
-	// this is a really problematic thing, because it breaks DNS resolution and IP routing, but its for an alpha feature that
-	// I need to work cluster-up
-	kubePluginInitializer.SetServiceResolver(aggregatorapiserver.NewClusterIPServiceResolver(
-		informers.GetClientGoKubeInformers().Core().V1().Services().Lister(),
-	))
+		generic.NewConfiguration(quotaRegistry.List(), map[schema.GroupResource]struct{}{}))
+
+	webhookInitializer := webhookinitializer.NewPluginInitializer(
+		func(delegate webhookconfig.AuthenticationInfoResolver) webhookconfig.AuthenticationInfoResolver {
+			return webhookconfig.AuthenticationInfoResolverFunc(func(server string) (*rest.Config, error) {
+				if server == "kubernetes.default.svc" {
```

Comment:
after I was only able to get all of this to work in local-cluster-up.sh we really have to test webhooks on origin. I fear we are also hit by the broken loopback client config mutation problems that are deep in the webhook code. Adding a followup